.github/workflows/compilation.yml

```diff
@@ -43,5 +45,5 @@ jobs:
           automatic_release_tag: "latest"
           title: Development build
           files: |
-            iopmod-modules-${{ github.sha }}
+            iopmod-modules.tar.gz
```

Comment:
I propose that we sprinkle the file name with tags such as `ubuntu` and `x86-64`. Alternatively, much better, we could build [statically linked](https://en.wikipedia.org/wiki/Static_library) tools. Such tools would work with _all_ Linux/amd64 variants (Fedora, Arch, Debian, Slackware, and so on and so forth), as they are independent of libraries and only use Linux kernel system calls.

Sadly, the tools still won’t work for people on [POWER](https://en.wikipedia.org/wiki/IBM_POWER_architecture) and [RISC-V](https://en.wikipedia.org/wiki/RISC-V) and other free architectures. Unless we apply the Gentoo [crossdev](https://wiki.gentoo.org/wiki/Crossdev) package again, not only for the R3000 modules, but for these as well... :-)